axlearn/cloud/common/quota_test.py

```diff
@@ -17,8 +17,8 @@
         "resource_type2": 8,
     },
     "project_resources": {
-        "team1": {"resource_type1": 0.5},
-        "team2": {"resource_type1": 0.5, "resource_type2": 1.0},
+        "team1": {"resource_type1": 0.3},
```

Comment:
Does this test fail without the change? I get:

```
> 4.8+11.2==16
True
> 12.8+11.2==24
True
> 0.3+0.7==1
True
```

Maybe it would make sense to use:
```
> 0.1 + 0.2 <= 0.3
False
```

Reply:
Good point. Found a way to make a test. PTAL?